pkg/ui/workspaces/cluster-ui/src/insights/workloadInsights/util/insightsColumns.tsx

```diff
@@ -44,115 +48,112 @@ export function getLabel(
     let tooltipText = `The ${execType} query.`;
     if (execType == InsightExecEnum.TRANSACTION) {
       tooltipText = "The queries attempted in the transaction.";
     }
-    return (
-      <Tooltip style="tableTitle" placement="bottom" content={tooltipText}>
-        {getLabel("query", execType)}
-      </Tooltip>
-    );
+    return makeToolTip(<p>tooltipText</p>, "query", execType);
   },
   insights: (execType: InsightExecEnum) => {
-    return (
-      <Tooltip
-        placement="bottom"
-        style="tableTitle"
-        content={
-          <p>
-            The category of insight identified for the {execType} execution.
-          </p>
-        }
-      >
-        {getLabel("insights")}
-      </Tooltip>
+    return makeToolTip(
+      <p>The category of insight identified for the {execType} execution.</p>,
+      "insights",
     );
   },
   startTime: (execType: InsightExecEnum) => {
-    return (
-      <Tooltip
-        style="tableTitle"
-        placement="bottom"
-        content={<p>The timestamp at which the {execType} started.</p>}
-      >
-        {getLabel("startTime")}
-      </Tooltip>
+    return makeToolTip(
+      <p>The timestamp at which the {execType} started.</p>,
+      "startTime",
     );
   },
   elapsedTime: (execType: InsightExecEnum) => {
-    return (
-      <Tooltip
-        style="tableTitle"
-        placement="bottom"
-        content={
-          <p>The time elapsed since the {execType} started execution.</p>
-        }
-      >
-        {getLabel("elapsedTime")}
-      </Tooltip>
+    return makeToolTip(
+      <p>The time elapsed since the {execType} started execution.</p>,
+      "elapsedTime",
     );
   },
   username: (execType: InsightExecEnum) => {
-    return (
-      <Tooltip
-        style="tableTitle"
-        placement="bottom"
-        content={<p>The user that opened the {execType}.</p>}
-      >
-        {getLabel("username")}
-      </Tooltip>
-    );
+    return makeToolTip(<p>The user that opened the {execType}.</p>, "username");
   },
   applicationName: (execType: InsightExecEnum) => {
-    return (
-      <Tooltip
-        style="tableTitle"
-        placement="bottom"
-        content={<p>The name of the application that ran the {execType}.</p>}
-      >
-        {getLabel("applicationName")}
-      </Tooltip>
+    return makeToolTip(
+      <p>The name of the application that ran the {execType}.</p>,
+      "applicationName",
     );
   },
   numRetries: () => {
-    return (
-      <Tooltip
-        style="tableTitle"
-        placement="bottom"
-        content={"The number of times this statement encountered a retry."}
-      >
-        {getLabel("numRetries")}
-      </Tooltip>
+    return makeToolTip(
+      <p>The number of times this statement encountered a retry.</p>,
+      "numRetries",
+    );
+  },
+  isFullScan: () => {
+    return makeToolTip(
+      <p>
+        The table is scanned on all key ranges of the rides_pkey index (i.e., a
```

Comment:
looks like this is specific to an example, with a table called rides.
You cna change to "The statement executed a full scan."